src/main/java/nl/tudelft/ti2206/group9/gui/GameWindow.java

```diff
@@ -40,10 +49,18 @@
 	private static SubScene overlayScene;
 	private static ExternalTicker extTicker;
 	private static boolean running;
+	private static Stage primaryStage;
+
+	
+	/** Hide public constructor. */
+	private GameWindow() { }
 
 	/** Start the Application. */
-	public static void start(Stage primaryStage) {
+	public static void start(Stage stage) {
 		State.resetAll();
```

Comment:
Hey @thervh70 , I left this line here because this was the starting point of the Application. I think this line should be moved to SplashScreen, and that the deathMenu restart button should call `reset()` (saving the player's total amount of coins, `resetAll()` also resets coins).